src/mappings/curation.ts

```diff
@@ -103,10 +103,7 @@ export function handleBurned(event: Burned): void {
   deployment.signalAmount = deployment.signalAmount.minus(event.params.signal)
   deployment.save()
 
-  // Update epoch
-  let epoch = createOrLoadEpoch(event.block.number)
```

Comment:
Removed unsignaled tokens. the field should just accumulate signaled tokens in an epoch